pkg/arn/arn.go

```diff
@@ -0,0 +1,104 @@
+package arn
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+// ARN - Custom ARN type for helping with formatting
+type ARN struct {
+	arn.ARN
```

Comment:
Nesting the aws arn struct.  Adding functions to make working with ARNs in and out of json and dynamodb easier.